packages/table/test/tableTests.tsx

```diff
@@ -833,6 +882,7 @@ describe("<Table>", () => {
 
             const viewportRect = table.state("viewportRect");
             expect(viewportRect.left).to.equal((NUM_COLS * UPDATED_COL_WIDTH) - viewportRect.width);
+            expect(onVisibleCellsChange.callCount).to.be.at.least(3);
```

Comment:
is it inconsistent? If so, that seems like an issue. If it's consistent, just do an equals check. Tighten those bounds!